xlsxdocument/document.py

```diff
@@ -32,7 +32,7 @@ def table(self, titles, rows):
             processed = []
             for i, value in enumerate(row):
                 if isinstance(value, dt.datetime) and hasattr(value, "utcoffset"):
```

Comment:
```suggestion
                if isinstance(value, dt.datetime):
```

funktionierts so auch? falls ja würde ich ein release veröffentlichen.

Reply:
@indro

Reply:
Nope, kommt gleicher Fehler.